api/api.proto

```diff
@@ -156,14 +156,11 @@ message BolusAmountMaximumGuardRail {
   // Bolus units
   BolusUnits units = 1;
 
-  // Default value
-  FixedDecimal defaultValue = 2;
```

Comment:
You should also mark the field as reserved so we don't reuse it by accident https://developers.google.com/protocol-buffers/docs/proto3#reserved

Reply:
Done.  Thanks for pointing this out.